graylog2-web-interface/src/views/pages/ExtendedSearchPage.test.jsx

```diff
@@ -261,4 +262,17 @@ describe('ExtendedSearchPage', () => {
         expect(SearchActions.execute).not.toHaveBeenCalled();
       });
   });
+
+  it('information about execution error will be displayed', async () => {
+    asMock(SearchActions.execute).mockImplementationOnce(() => Promise.reject(new Error('The error message.')));
+    let wrapper;
+    await act(async () => {
+      wrapper = mount(<SimpleExtendedSearchPage />);
+    });
+
+    if (wrapper) {
+      wrapper.update();
+      expect(wrapper.contains('The error message.')).not.toBeNull();
+    }
+  });
```

Comment:
I received an `act()` warning because of the update triggered by throwing the error. How do we deal with this case normally when using enzyme? There has to be a more elegant way.

Reply:
This comment is no longer relevant, because we've adjusted the error handling.